--- flawfinder-0.20/flawfinder~	Sat Dec 22 13:13:10 2001
+++ flawfinder-0.20/flawfinder	Sat Dec 22 13:13:10 2001
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python
 
 # flawfinder: Find potential security flaws ("hits") in source code.
 # Usage:
--- flawfinder-0.20/setup.py~	Sat Dec 22 13:13:37 2001
+++ flawfinder-0.20/setup.py	Sat Dec 22 13:13:37 2001
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python
 
 # This is the setup.py script for "flawfinder" by David A. Wheeler.
 # My thanks to Jon Nelson, who created the initial setup.py script.
